common/utils/src/main/scala/org/apache/spark/util/MavenUtils.scala

```diff
@@ -432,7 +443,7 @@ private[spark] object MavenUtils extends Logging {
         printStream.println(s"The jars for the packages stored in: $packagesDirectory")
         // scalastyle:on println
 
-        val ivy = Ivy.newInstance(ivySettings)
+        var ivy = Ivy.newInstance(ivySettings)
```

Comment:
We can give it a new name below, no need to change it to 'var' here.

Reply:
Done